plugins/modules/postgresql_info.py

```diff
@@ -731,22 +731,22 @@ def get_tablespaces(self):
                               "AND column_name = 'spcoptions'")
 
         if not opt:
-            query = ("SELECT s.spcname, pg_catalog.pg_get_userbyid(s.spcowner) as rolname, s.spcacl "
+            query = ("SELECT s.spcname, pg_catalog.pg_get_userbyid(s.spcowner) as rolname, s.spcacl::varchar "
```

Comment:
why varchar and not text? the field is of type aclitem and in pg varchar is just text w/ input limits

Reply:
it probably doesn't make a difference, i'm just curious.

Reply:
I wasn't sure which one to use, whether text or varchar, so picked varchar by habit. 
Happy to change it, shouldn't make a difference.